src/test/java/io/lakefs/iceberg/TestCatalogMigration.java

```diff
@@ -10,10 +10,11 @@
 import org.apache.spark.sql.types.StructField;
 import org.apache.spark.sql.types.StructType;
 import org.junit.Test;
-import static org.junit.Assert.assertEquals;
 
+import java.util.Collections;
```

Comment:
Not in this PR: Naming is very strange here.  I know only the JUnit naming convention, _`ClassName`_`Test`, as [here](https://github.com/junit-team/junit4/wiki/Getting-started#create-a-test).  All style guides I've seen also say to use this one, e.g. the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names).

Starting with `Test` makes autocompletion and search and others much less useful:
* Classes don't show up with their tests (prefixes are usually preferred, and sorting puts all the tests somewhere in the middle on the letter "T").
* Narrow windows show much less information when looking at tests, because the first 4 characters are always "Test" and provide no information.